lib/Ethereum/RPC/Client.pm

```diff
@@ -2,17 +2,20 @@ package Ethereum::RPC::Client;
 
 use strict;
 use warnings;
+
 use Moo;
-use MojoX::JSON::RPC::Client;
+use HTTP::Request;
+use JSON::MaybeXS;
+use LWP::UserAgent;
```

Comment:
I think Mojo::UserAgent is preferred choice so may be use that? then I think you don't need HTTP::Request in that case